projects/coolstore-kjar-s2i/Jenkinsfile

```diff
@@ -2,52 +2,39 @@ node {
     
     def WORKSPACE = pwd()
     def mvnHome = tool 'maven'
+    def token = readFile('/var/run/secrets/kubernetes.io/serviceaccount/token')
+    def ruleVersion = "${RULES_VERSION}" != "" ? "${RULES_VERSION}" : "2.0.0"
    
    stage 'Checkout'
 
        checkout scm
        
-  stage 'Update Rules'
+  stage 'Retrieve Rules'
 
        sh """
-           rm -rf rules-clone
-           mkdir ${WORKSPACE}/rules-clone
-           pushd ${WORKSPACE}/rules-clone
-           git clone git://10.3.8.92:9418/coolstore
-           cd coolstore/coolstore
-           mvn clean install
-           popd
-           rm -rf project-clone
-           mkdir project-clone
-           pushd project-clone
-           git clone git@github.com:sabre1041/ose-bdd-coolstore-demo.git
-           cd ose-bdd-coolstore-demo
-           cp -f ${WORKSPACE}/rules-clone/coolstore/coolstore/target/coolstore* projects/coolstore-kjar-s2i/deployments
-           git add projects/coolstore-kjar-s2i/deployments/coolstore*
-           git commit -m "jenkins - update to rules"
-           git push origin master
-           popd
+           cd ${WORKSPACE}/projects/coolstore-kjar-s2i
+           ${mvnHome}/bin/mvn clean org.apache.maven.plugins:maven-dependency-plugin:copy -Dartifact=com.redhat:coolstore:${ruleVersion} -Dmdep.stripVersion=true -DoutputDirectory=target/s2i-build
        """
 
 
     stage 'OpenShift Build'
       
         try {
-            // Download support scripts
-            sh "mkdir -p ${WORKSPACE}/build-tools && curl -s -o ${WORKSPACE}/build-tools/ose-build-monitor.sh https://raw.githubusercontent.com/sabre1041/ose3-samples/master/trigger-monitor-build.sh && chmod +x ${WORKSPACE}/build-tools/ose-build-monitor.sh && curl -s -o ${WORKSPACE}/build-tools/ose-deploy-monitor.sh https://raw.githubusercontent.com/sabre1041/ose3-samples/master/verify-deploy.sh && chmod +x ${WORKSPACE}/build-tools/ose-deploy-monitor.sh"
-   
-            withCredentials([[$class: 'StringBinding',
-                credentialsId: 'bdd-ose-token', variable: 'OSE_TOKEN']]) {
-                
-                    sh """
-
-                        ${WORKSPACE}/build-tools/ose-build-monitor.sh -h=${OPENSHIFT_HOST} -t=${env.OSE_TOKEN} -n=${OPENSHIFT_SOURCE_PROJECT} -a=${OPENSHIFT_SOURCE_APPLICATION}
-                   
-                        ${WORKSPACE}/build-tools/ose-deploy-monitor.sh -h=${OPENSHIFT_HOST} -t=${env.OSE_TOKEN} -n=${OPENSHIFT_SOURCE_PROJECT} -a=${OPENSHIFT_SOURCE_APPLICATION}
+            // Prepare Artifacts
+            sh """
+                mkdir -p ${WORKSPACE}/projects/coolstore-kjar-s2i/target/s2i-build/deployments
+                mv ${WORKSPACE}/projects/coolstore-kjar-s2i/target/s2i-build/coolstore.jar ${WORKSPACE}/projects/coolstore-kjar-s2i/target/s2i-build/deployments/ROOT.jar
```

Comment:
This should not be moved to root.war.  It should stay as a jar named as is.  It is not putting it in standalone/deployments, if it is built with the kie maven plugin which it is, it will be put in the local m2 of the kie server image.